### PR TITLE
Inertia tensor

### PR DESCRIPTION
**Pull request by JegerBroxterman**
Included reduced inertia and reduced projected inertia tensors. Changed the inertia tensor calculation by dividing by the total mass.

### CODE REVIEW

File: kinematic_properties.py [hunk 1]
```diff
@@ -132,7 +132,7 @@ def get_vmax(mass, radius):
 def get_inertia_tensor(mass, position):
 
     # 3x3 inertia tensor
-    Itensor = (mass[:, None, None] * position[:, None:, None] * position[:, None]).sum(
+    Itensor = (mass[:, None, None]/mass.sum() * position[:, None:, None] * position[:, None]).sum(
```
Comment (robjmcgibbon) [inline kinematic_properties.py:135]: Why are you changing the units of the inertia tensor?Do you know of anyone else who uses this property that will need to be informed of the change?

File: kinematic_properties.py [hunk 2]
```diff
@@ -171,10 +171,64 @@ def get_projected_inertia_tensor(mass, position, axis):
         np.array((Itensor[0, 0], Itensor[1, 1], Itensor[0, 1]))
         * position.units
         * position.units
-        * mass.units
     )
     return Itensor
 
+def get_reduced_inertia_tensor(mass, position):
```
Comment (robjmcgibbon) [inline kinematic_properties.py:177]: There is quite a lot of duplicated code shared with get_inertia_tensor. Could you combine these functions by passing a keyword argument, e.g. `get_inertia_tensor(mass, position, reduce=False)`

File: kinematic_properties.py [hunk 3]
```diff
@@ -171,10 +171,64 @@ def get_projected_inertia_tensor(mass, position, axis):
...
+    # 3x3 inertia tensor
+    Itensor = (mass[:, None, None]/mass.sum() * position[:, None:, None] * position[:, None] / norm[:, None, None]).sum(
+        axis=0
+    )
+
+    # Symmetric, so only return lower triangle
+    Itensor = np.concatenate([np.diag(Itensor), Itensor[np.triu_indices(3, 1)]])
+    return Itensor
+
+def get_reduced_projected_inertia_tensor(mass, position, axis):
```
Comment (robjmcgibbon) [inline kinematic_properties.py:193]: You could also pass a keyword argument to `get_projected_inertia_tensor`

File: kinematic_properties.py [hunk 4]
```diff
@@ -171,10 +171,64 @@ def get_projected_inertia_tensor(mass, position, axis):
...
+    # Note: unyt currently ignores the position units in the *=
+    # i.e. Itensor is dimensionless throughout (even though it should not be)
+    for i in range(2):
+        for j in range(2):
+            Itensor[:, i, j] *= (
+                projected_position[:, i].value * projected_position[:, j].value / norm.value
+            )
+    Itensor = Itensor.sum(axis=0)
+    Itensor = (
+        np.array((Itensor[0, 0], Itensor[1, 1], Itensor[0, 1]))
```
Comment (robjmcgibbon) [inline kinematic_properties.py:225]: Do you need to multiply by units here?

**Review by robjmcgibbon — COMMENTED** [review]
Added a couple of comments. Also could you please run the formatter?

File: kinematic_properties.py (hunk 2 above)
Comment (JegerBroxterman) [reply kinematic_properties.py:177]: Yes, I will make the changes

**Pull request merged**